documentation/decisions/adr/2023-09-07-data-replication-tool.md

```diff
@@ -0,0 +1,94 @@
+# Data Replication Strategy & Tool
+
+- **Status:** Active
+- **Last Modified:** 2023-09-07
+- **Related Issue:** [#322](https://github.com/HHS/grants-equity/issues/322)
+- **Deciders:** Lucas Brown, Billy Daly, Sammy Steiner, Daphne Gold, Aaron Couch
+- **Tags:** Hosting, Infrastructure, Database
+
+## Context and Problem Statement
+
+The Beta.Grants.Gov platform will need to consume data and apply additional load to the database, that the database was not planned to support. Therefore, we will replicate the data for the Beta.Grants.Gov work so it will have a negligable impact on the database for replication related tasks and still provide up-to-date production data.
+
+## Decision Drivers
+
+- Data source and destination compatibility: rep tool should support the data sources used in the project (db, file systems) and is compatible with the target destination (db, warehouses, cloud storage).
+- Data volume and throughput: tool can handle the volume and throughput requirements of the data replication process efficiently.
+- Data transformation capabilities: replication tool can handle data transformation during the replication process, including data format conversions and schema changes.
+- Real-time vs. batch replication: determine whether real-time data replication or if batch replication at scheduled intervals is sufficient.
+- Latency and performance: consider the latency and performance to ensure timely data updates and minimal impact on system performance.
+- Security and encryption: replication tool provides adequate security features, including data encryption and secure data transmission.
+- Monitoring and alerting: to promptly identify and address replication issues.
+- Ease of use and configuration: Evaluate the tool's user-friendliness and ease of configuration, as complex setup processes can lead to inefficiencies.
+- Scalability: Determine if the replication tool can scale to accommodate future growth and increased data demands.
+- Cost: Consider the licensing and operational costs
+- Support and community: Assess availability of support options and the size and activity of the tool's user community.
+
+## Options Considered
+
+- Use Production Database
+- Use AWS DMS (Database Migration Service)
+- Create new data pipelines from data sources
+
+## Decision Outcome
+
+Chosen option: Use AWS DMS, because it is the only option that allows us to deliver within our period of performance and doensn't impact the production database' ability to perform its existing role.
```

Comment:
nit: a couple typos (`doensn't`, `database'`)

Reply:
Thanks! went through the doc again for typos.